README.md

```diff
@@ -546,9 +546,11 @@ The following environment variables are respected by `request`:
 
  * `HTTP_PROXY` / `http_proxy`
  * `HTTPS_PROXY` / `https_proxy`
+ * `ALL_PROXY` / `all_proxy`
  * `NO_PROXY` / `no_proxy`
 
 When `HTTP_PROXY` / `http_proxy` are set, they will be used to proxy non-SSL requests that do not have an explicit `proxy` configuration option present. Similarly, `HTTPS_PROXY` / `https_proxy` will be respected for SSL requests that do not have an explicit `proxy` configuration option. It is valid to define a proxy in one of the environment variables, but then override it for a specific request, using the `proxy` configuration option. Furthermore, the `proxy` configuration option can be explicitly set to false / null to opt out of proxying altogether for that request.
+If neither `http_proxy` nor `https_proxy` is set, then `all_proxy` is used if available.
```

Comment:
Do you really see value in stating the obvious that if no proxy config is set, that the requests are not proxied?